lib/Redmine/Api/AbstractApi.php

```diff
@@ -96,7 +96,7 @@ protected function delete($path)
      */
     protected function isNotNull($var)
     {
-        return !is_null($var);
+        return !(is_null($var) || $var === false || $var === "" || ((is_array($var) || is_object($var)) && empty($var)));
```

Comment:
Could we make this the other way around so it is easier to read/understand?
Would that work? 
'''
false !== $var &&
null !== $var &&
'' !== $var &&
( (is_array($var) || is_object($var)) && !empty($var) )
'''